File: swagger/models/get_client_indexes_response.py
from .base_model import BaseModel

from .client_index import ClientIndex

class GetClientIndexesResponse(BaseModel):
	"""
	no description available

	Attributes:

		client_indexes
		Contains information about the client indexes.
	"""

	input_map = {
		'ClientIndexes': ['client_indexes', ClientIndex],
		}

	output_map = {
		'client_indexes': 'ClientIndexes',
		}

	client_indexes = None

File: swagger/Models/GetClientIndexesResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetClientIndexesResponse extends BaseModel
{
	/**
	 * Contains information about the client indexes.
	 * 
	 * @var ClientIndex[] 
	 */
	public $clientIndexes;

	protected function getInputMap()
	{
		return [
			'ClientIndexes' => ['clientIndexes', 'array', ClientIndex::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'clientIndexes' => ['ClientIndexes', 'array', ClientIndex::class],
			];
	}
}
